### DISCUSSION TITLE
House category is missing

### DISCUSSION THREAD

**Discussion by cp76** [streetcomplete/StreetComplete#5386] [Feedback]
I just started using the app and I really like it!

I initially started by looking for tasks in my immediate neighborhood and answering the questions. In doing so, I encountered a problem. 
There is a building here that does not fit into any of the house categories you can choose from in the German version of the app. It is a so-called "Aerztehaus" which directly translates into doctor's house or more generalized medical center. This is a multi-story building that houses one or more doctor's offices. Can such a category be added in future versions?

Thanks a lot for your great work!

**Comment by mnalis** [comment]
I don't know about German translation, but it sounds like something that I'd probably tag as _"Hospital building"_ (under _"Civic building"_ category). If you feel that is not how the building looks, you can choose more general _"Civic building"_ category itself.

Note that this quest is only about _**architectural type of the building**_ and **not** about _**what it is used for**_, so being super-precise is not as important. To mark actual things that people might look for (i.e. amenities like doctors, shops, fuel stations etc.) you need to use **"Shops overlay"** in StreetComplete (but even there not everything is available yet, see https://github.com/streetcomplete/StreetComplete/issues/5152 for details if interested).
 
See [StreetComplete FAQ - Building Type Quest - Why don't I see the right types?](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Building_Type_Quest_-_Why_don't_I_see_the_right_types?) for more detailed explanation.

**Comment by HolgerJeromin** [comment]
Some doctor houses are quite similar to apartment buildings (at least in Germany).
Small separate parts with each multiple rooms in it.
They are also often mixed (residential + doctors) or parts changing between both uses.

Hospitals have different structure inside: more central reception point, large wide floors.

**Reply by mnalis** [reply]
> Some doctor houses are quite similar to apartment buildings (at least in Germany).

Ah, thanks for clarification. I unfortunately have no experience with how it looks exactly in Germany, so I can only give descriptional advice, not exact answer. If it looks like an apartment building (e.g. because it was built as such before parts of it were repurposed) than I agree with you it would indeed better be marked as _"apartment building"_.

>  They are also often mixed (residential + doctors) or parts changing between both uses.

For multiple-purpose building, one might do as the `Other answers`  / `It has multiple purposes` say: simply select the main (majority) purpose. (so if it is e.g. mostly apartment building with few doctor offices, mark it as _"apartment building"_)

> Hospitals have different structure inside: more central reception point, large wide floors.

I agree. I was going with _"... more generalized medical center"_ part of the explanation which sounded more like hospital to me. Do mark it as apartment building if it looks more like one!

**Reply by westnordost** [reply]
Or, office buildings.
